nav2_behavior_tree/include/nav2_behavior_tree/plugins/action/compute_path_through_poses_action.hpp

```diff
@@ -86,7 +86,7 @@ class ComputePathThroughPosesAction
           "Mapped name to the planner plugin type to use"),
         BT::OutputPort<nav_msgs::msg::Path>("path", "Path created by ComputePathThroughPoses node"),
         BT::OutputPort<ActionResult::_error_code_type>(
-          "compute_path_through_poses_error_code", "The compute path through poses error code"),
```

Comment:
Shouldn't the NavigateToPose and NavigateThroughPoses BT Nodes also have the output port for the error code to set if there are autonomy applications using them (since the piping now exposes those return codes through to the action's caller)?

Reply:
I don't think you need to do anything with them other than exposing the output port and setting the error code in those 2 BT nodes. We don't have any integration tests using these nodes that would need updating and we already show that the Nav2Pose action pipes correctly  from your WPF test updates so this can just be done in code and we're good

Reply:
Named navigate_to_pose_error_code + navigate_through_poses_error_code